src/core/DataFile.cpp

```diff
@@ -868,10 +869,21 @@ void DataFile::upgrade_1_1_91()
 	for( int i = 0; !list.item( i ).isNull(); ++i )
 	{
 		QDomElement el = list.item( i ).toElement();
-		if ( el.attribute( "name" ) == "plugin" && el.attribute( "value" ) == "vocoder-lmms" ) {
+		if( el.attribute( "name" ) == "plugin" && el.attribute( "value" ) == "vocoder-lmms" ) {
 			el.setAttribute( "value", "vocoder" );
 		}
 	}
+
+	list = elementsByTagName( "crossoevereqcontrols" );
+	for( int i = 0; !list.item( i ).isNull(); ++i )
+	{
+		QDomElement el = list.item( i ).toElement();
+		// invert the mute LEDs
+		for( int d = 1; d <= 4; ++d ){
```

Comment:
Ahh, I did something differerent, and now that I think about it I can absolutely understand why it didn't work :blush: @tresf thanks for the info on the proper logic, but I'll just keep this PR as-is. @grejppi if you could test this out, it'd be great.